content/concepts/peers.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Peer Identity
+title: All about Peers
 weight: 4
 ---
 
+### What is a PeerId
+
 A Peer Identity (often written `PeerId`) is a unique reference to a specific
```

Comment:
Can we just write them as Peer ID? There's no need to pretend that it's a variable.
(I'm fine if that's done in a separate PR).

Reply:
Yes